pinot-common/src/main/java/com/linkedin/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -39,13 +39,15 @@
  *
  * Supports serialization via JSON.
  */
-@JsonPropertyOrder({"selectionResults", "traceInfo", "numDocsScanned", "aggregationResults", "timeUsedMs", "segmentStatistics", "exceptions", "totalDocs"})
+@JsonPropertyOrder({"selectionResults", "aggregationResults", "exceptions", "numDocsScanned", "numEntriesScannedInFilter", "numEntriesScannedPostFilter", "totalDocs", "timeUsedMs", "segmentStatistics", "traceInfo"})
```

Comment:
We are not using ObjectMapper to read this class, so IMO we don't need ignore unknown fields.